packages/api/src/lib/prepareParams.ts

```diff
@@ -131,7 +148,7 @@ module.exports = async (operation, body, metadata) => {
         // If more than 25% of the body intersects with the parameters that we've got on hand, then we should treat it
         // as a metadata object and organize into parameters.
         // eslint-disable-next-line no-param-reassign
-        metadata = body;
+        metadata = body as Record<string, unknown>; // @todo we should just do a "if body isnt an object don't digest it"
```

Comment:
This is a potential bug I found while typing everything. Will handle and add tests for it in another PR.